lib/fog/vsphere/models/compute/cdrom.rb

```diff
@@ -0,0 +1,65 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdrom < Fog::Model
+        identity :key
+
+        attribute :filename
+        attribute :name, :default => "CD-/DVD-ROM Drive"
+        attribute :controller_key
+        attribute :unit_number
+        attribute :start_connected
+        attribute :allow_guest_control
+        attribute :connected
+
+        has_one_identity :server, :servers, :aliases => :instance_uuid
+
+        def to_s
+          name
+        end
+
+        def destroy
+          requires :server_id, :key, :unit_number
```

Comment:
I guess this will break since the attribute `server_id` was removed. Can you confirm?